### PR TITLE
Added Odia language to language selector 

### LINKED ISSUE #9393: Odia translation completed
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

I have completed Odia translation using the weblate. Add "Odia" language to Newpipe.

### Why do you want this feature?

Odia is the mother tongue of about 10 crores of people.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by GET100PERCENT**
#### What is it?
- [ ] Bugfix (user facing)
- [x] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

It adds the Odia language to the language selector.

#### Fixes the following issue(s)

It solves the issue #9393.
- Fixes #9393 

#### APK testing
I have downloaded and the zip file from the artifacts available in the changes section and opened the apk file inside the zip and installed it.
It works perfectly, odia language is available in the language selector in this apk file.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by GET100PERCENT** [comment]
Screenshot_20230109-192810
Screenshot_20230109-192737
The above are Screenshots before and after the Odia language has been added to the language selector.

**Comment by GET100PERCENT** [comment]
> Fill the template properly.

Hi, i have updated the template.

**Review by Stypox — APPROVED** [review]
Thanks! (will let @TobiGr merge)

File: app/src/main/res/values/settings_keys.xml [hunk 1]
```diff
@@ -523,6 +524,7 @@
         <item>Nynorsk</item>
         <item>O‘zbek</item>
         <item>Polski</item>
+        <item>ଓଡ଼ିଆ</item>
```
Comment (TobiGr) [inline app/src/main/res/values/settings_keys.xml:527]: You have swapped Polish and Odia. Please move this line up.

**Review by TobiGr — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/settings_keys.xml (hunk 1 above)
Comment (GET100PERCENT) [reply app/src/main/res/values/settings_keys.xml:527]: Okay

Comment (GET100PERCENT) [reply app/src/main/res/values/settings_keys.xml:526]: > You have swapped Polish and Odia. Please move this line up.

I have performed the changes suggested by you.

**Review by TobiGr — APPROVED** [review]

**Pull request merged**